app/models/piece.rb

```diff
@@ -18,12 +17,53 @@ def move!(x, y)
     true
   end
 
+  # First check if coordinates is taken
+  # return false if coordinates belongs to friendly piece
+  # otherwise return true
+  # return false if position is not taken
+  def capturable?(x, y)
+    if position_taken?(x, y)
+      return false if friendly_piece?(x, y)
+      true
+    else
+      false
+    end
+  end
+
+  # if piece is capturable change piece's attributes
+  def capture!(x, y)
+    if capturable?(x, y)
+      target_piece(x, y).update_attributes!(
+        x_position: nil,
+        y_position: nil,
+        captured: true
+      )
+    end
+  end
```

Comment:
Can eliminate a line here by converting the conditional into a guard clause.  Guard clause would look like `return false unless capturable?(x, y)`.